packages/legend-query/src/stores/QueryBuilderPostFilterOperator.ts

```diff
@@ -34,7 +34,13 @@ export abstract class QueryBuilderPostFilterOperator {
 
   abstract getLabel(): string;
 
-  abstract getPureFunction(): SUPPORTED_FUNCTIONS | undefined;
+  abstract getPureFunction(
```

Comment:
These 2 functions `getPureFunction` and `getPureFunctionFromExpression` nolonger makes sense, let's remove them and do it the same way we do `QueryBuilderFilterOperator`. Thanks!